synonymdev/bitkit-react-native
# fix: improve auto read clipboard UX

### Description

- adds a dialog prompt before auto-redirecting on valid clipboard data
- add OS version to support email
- redirect to add contact on ActivityDetail if contact is not added yet
- fix layout shift in Send result screen
- fix no confirm text for activity with fee below minimum

### Type of change


### Tests


### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/70a58614-ab55-4187-ba3a-d4c347983e54

### QA Notes

Check with and without PIN code set, app starting fresh or opened from background.
